popde/utils_plot.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+
+def simple2Dplot(xx, yy, kde):
+    """
+    return kde plot in 2D given output
+    """
+    fig = plt.figure()
+    ax = fig.gca()
+    # Contourf plot
+    cfset = ax.contourf(xx, yy, kdevals, cmap='Blues')
+    # Contour plot
+    cset = ax.contour(xx, yy, kdevals, colors='k')
+    # Label plot
+    ax.clabel(cset, inline=1, fontsize=10)
+    ax.set_xlabel('Y1')
```

Comment:
why would you want x and y labels called 'Y1' and 'Y0' ?  The dimensions of the KDE already have names assigned to them, those should be used

Reply:
Or just leave it to the plotting method inside density_estimate.py to set the labels ..